client/src/Event/components/EventList.tsx

```diff
@@ -11,15 +12,18 @@ import { Spinner, SearchBar, Pagination } from 'common/components'
 import { numberWithCommas } from 'common/helpers'
 import useMediaQuery from 'common/hooks/useMediaQuery'
 import ExportButton from 'common/components/ExportButton'
+import EventListFilter from './EventListFilter'
 
 const EventList: FC = () => {
   const [currentPage, setCurrentPage] = useState(1)
   const [lastCursor, setLastCursor] = useState<string | undefined>(undefined)
   const PAGE_SIZE = 10
   const isDesktop = useMediaQuery('(min-width: 640px)')
+  const [timeDimension, setTimeDimension] = useState<'date' | 'block'>('block')
```

Comment:
Probably this state should be inside `EventListFilter` because it is not used anywhere in this component